src/Components/Facility/HospitalList.tsx

```diff
@@ -98,7 +98,7 @@ export const HospitalList = (props: any) => {
         facility_type: qParams.facility_type,
         kasp_empanelled: qParams.kasp_empanelled,
       };
-
+      if (params.search_text) params.offset = 0;
```

Comment:
@GokulramGHV this only works for the `search by name` filter and will not work for others, so why not add a page parameter as 1 when we make a request related to filters

Reply:
@GokulramGHV also checkout #4009
The page offset still remains to 2 :/

![image](https://user-images.githubusercontent.com/25143503/200807284-e4df7372-680c-42ce-abad-e63096757fd1.png)

cc: @khavinshankar

Reply:
@GokulramGHV @khavinshankar @nihal467 the changes suggested here are fixed in #4017. 
Merging this PR to develop will have no effect once #4017 is merged. since the reset method used in this PR is not based on query parameters. The changes were overridden in #4017